packages/nextjs/.env.example

```diff
@@ -1,3 +1,15 @@
+# PWA environment variables.
+# This all lives in the `.env` file, which is not committed to git
+PUBLIC_KEY_VAPID=
+PRIVATE_KEY_VAPID=
```

Comment:
Maybe we can moved this after the Firebase ones... since we fill them last if we follow the README workflow.

Reply:
fiexed at https://github.com/BuidlGuidl/PWA-burner-wallet/pull/1/commits/43e5b9acb94929128095266192facb3e43893c67